### PR TITLE
Multiple Fixes

### LINKED ISSUE #61: [BORKED] drop_table error on survival
> **Error Message**:
`Tag <script[drop_table_data].list_keys[mob_level.].numerical> is invalid!`

> **Script Information**:
Script Name:  `drop_table`
Script Reference:  [`survival/./repo-link/global_systems/drop_tables.dsc`](https://github.com/AuroraInteractive/network-script-data/tree/master/denizen_scripts/survival/repo-link/global_systems/drop_tables.dsc#L84)

This need to:
- run only if MythicMob dies (dont use that event, though)
- drop from the drop table based on the MythicMob's level

### LINKED ISSUE #65: [BORK] Mobs still spawn in claims with them disabled.
> **Describe the bug**:
Mobs are spawning in claims with them disabled.

> **How To Reproduce**:
turn off mob spawns, and wait in claim,

> **Expected behavior**:
mobs to not spawn

> **Screenshots**:

image

> **Additional context**:
eggs spawn mobs as well as vanilla spawns occuring.

### LINKED ISSUE #73: [BORK] Grim not cycling to night time
> **Describe the bug**:
Grims area no longer sets to night time

> **How To Reproduce**:
Walk to grims area

> **Expected behavior**:
Should be night time.

> **Screenshots**:
image

### PR DESCRIPTION
**Pull request by Ph4i1ur3**
- Fixes #73 
- Fixes #61 
- Fixes #65 
- Implements #41 
- Implements #57

### CODE REVIEW

**Comment by Ph4i1ur3** [comment]
Squash this when Pulled, please.

File: denizen_scripts/survival/repo-link/spawn_scripts/grim_effects.dsc [hunk 1]
```diff
@@ -4,13 +4,13 @@ grim_area_effects:
     on player enters spawn_grim_cuboid:
       - cast blindness duration:40t no_ambient hide_particles
       - time player 14000
-      - execute as_server "stopsound <player.name> music" silent
+      - adjust <player> stop_sound
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/grim_effects.dsc:7]: This is currently a spigot bug that passed into Denizen again like it did previously, while `execute` is usually a bad usage for the Denizen equivalent command, this one doesn't work
```suggestion
      - execute as_server "stopsound <player.name> music" silent
```

File: denizen_scripts/survival/repo-link/spawn_scripts/grim_effects.dsc [hunk 2]
```diff
@@ -4,13 +4,13 @@ grim_area_effects:
...
       - wait 1t
       - playsound <player> sound:MUSIC_DRAGON pitch:1 volume:150 sound_category:master
       - playsound <player> sound:custom.death_graveyard sound_category:master volume:0.4 custom
     on player exits spawn_grim_cuboid:
       - cast blindness duration:40t no_ambient hide_particles
       - time player reset
-      - execute as_server "stopsound <player.name> music" silent
+      - adjust <player> stop_sound
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/grim_effects.dsc:14]: comment above
```suggestion
      - execute as_server "stopsound <player.name> music" silent
```

File: denizen_scripts/survival/repo-link/spawn_scripts/survival_rtp.dsc [hunk 3]
```diff
@@ -77,7 +77,6 @@ spawn_sound_effects_handler:
   type: task
   debug: false
   script:
-    #- adjust <player> stop_sound:music
-    - execute as_server "stopsound <player.name> music" silent
+    - adjust <player> stop_sound
```
Comment (BehrRiley) [inline denizen_scripts/survival/repo-link/spawn_scripts/survival_rtp.dsc:80]: comment above
```suggestion
      - execute as_server "stopsound <player.name> music" silent
```

**Pull request merged**